docs/root/_static/searchtools.js

```diff
@@ -0,0 +1,565 @@
+/*
+ * searchtools.js
+ * ~~~~~~~~~~~~~~~~
+ *
+ * Sphinx JavaScript utilities for the full-text search.
+ *
+ * :copyright: Copyright 2007-2020 by the Sphinx team, see AUTHORS.
+ * :license: BSD, see LICENSE for details.
```

Comment:
Since this is only docs, I believe it's fine to include a BSD licensed component in the repository (which is Apache2 at root and for any binaries). @caniszczyk for confirmation.

Reply:
Please let me know if this is problematic. I can try to provide our own implementation of searchtools.js.

Reply:
I think it's totally fine, just want to double check. See https://softwareengineering.stackexchange.com/questions/40561/is-bsd-license-compatible-with-apache for a totally non-authoritative answer.

Reply:
yes https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#approved-licenses-for-allowlist

Reply:
Thanks @caniszczyk